user/storage.md

```diff
@@ -13,29 +13,45 @@ The following material was adapted from the
 [Pangeo Cloud User Guide](https://pangeo.io/cloud.html)
 :::
 
-## Your Home Directory
+## The JupyterHub Filesystem
```

Comment:
```suggestion
## Your Home Directory
```

IMO this is more consistent with what we use everywhere else in JupyterHub world.

Reply:
@yuvipanda I added that section title intentionally to have what i thought were more consistent nested subtitles: 
```
## The JupyterHub Filesystem
### Your Home Directory
### The shared Directory
### The /tmp Directory
```

Reply:
My brain, why it no workey?